lib/main_common.pm

```diff
@@ -2581,6 +2581,9 @@ sub load_hypervisor_tests {
         ENABLE_SRIOV_NETWORK_CARD_PCI_PASSTHROUGH => {
             modules => ['virt_autotest/sriov_network_card_pci_passthrough'],
         },
+        ENABLE_SEV_SNP_GUEST_VERIFICATION => {
+            modules => ['virt_autotest/sev_snp_guest_verification'],
```

Comment:
Based on what this file does(host+guest checks for sev-snp) and our conventions for naming other feature tests, seems better to use below? 
```
        ENABLE_SEV_SNP => {
            modules => ['virt_autotest/sev_snp_validation'],
```

Reply:
why not use "VIRT_SEV_SNP_GUEST_INSTALL" directly?

Reply:
> why not use "VIRT_SEV_SNP_GUEST_INSTALL" directly?
if just use the same parameter VIRT_SEV_SNP_GUEST_INSTALL, sometimes it is confused.